### ISSUE TITLE
The Formless Void (major Essence) is simulated without minor

### ISSUE THREAD

**Issue by Bloodmallet** [simulationcraft/simc#5052]
### Describe the bug
When simulating the major effect of The Formless Void the actor doesn't seem to get the minor effect.

![image](https://user-images.githubusercontent.com/9075987/74594087-9e247e00-5032-11ea-8a0a-02899af45d53.png)

### Expected behavior
The major essence should always simulate at least as much as its minor.

### To Reproduce
```
calculate_scale_factors=0
default_actions=1
default_skill=1.0
fight_style=patchwerk
fixed_time=1
html=bfa_test.html
iterations=60000
# log=1
# output=log.txt
# debug=1
optimize_expressions=1
target_error=0.05
# threads=8
profileset_work_threads=2
chart_show_relative_difference=1
# ptr=1

#Tier25/T25_Shaman_Elemental.simc

# trinket1=,id=174500
# trinket2=,id=174528
#set_bonus=titanic_empowerment_2pc=1


shaman=baseline
spec=elemental
level=120
race=tauren
role=spell
position=ranged_back
talents=2311132
azerite_essences=4:3/5:3/32:3/35:3
head=,id=174151,bonus_id=4824/1517/4775,azerite_powers=416/457/30/15/13
neck=,id=158075,bonus_id=4929/5814/4936/1630,azerite_level=80
shoulders=,id=174149,bonus_id=4824/1517/4775,azerite_powers=457/416/30/44/13
back=,id=169223,bonus_id=6286
chest=,id=174150,bonus_id=4800/4786/1517/4775,azerite_powers=416/582/22/13
wrists=,id=174161,bonus_id=4824/1517
hands=,id=174162,bonus_id=4824/1517
waist=,id=174158,bonus_id=4824/1517/6554
legs=,id=174159,bonus_id=4824/1517
feet=,id=174160,bonus_id=4824/1517
finger1=,id=169159,bonus_id=4824/1522,enchant=accord_of_versatility
finger2=,id=169157,bonus_id=4824/1522,enchant=accord_of_versatility
trinket1=,id=174060,bonus_id=4824/1517
trinket2=,id=174103,bonus_id=4800/4786/1517
main_hand=,id=172198,bonus_id=4800/4786/1517/6570,enchant=machinists_brilliance
off_hand=,id=172188,bonus_id=4824/1517
azerite_essences=
name="baseline"

# Profileset start
ready_trigger=1
profileset."37_1_0"+=azerite_essences=37:1:0
profileset."37_1_1"+=azerite_essences=37:1:1
profileset."37_2_0"+=azerite_essences=37:2:0
profileset."37_2_1"+=azerite_essences=37:2:1
profileset."37_3_0"+=azerite_essences=37:3:0
profileset."37_3_1"+=azerite_essences=37:3:1
```

**Comment by AlexanderKenny** [comment]
This seems like the minor is always being received properly. Formless major has no implementation/dps gain so you'd see all the majors in error range of baseline if it were the case that no minor was inherited. What it looks like however is that when simming it as a 'major' it's always treated as rank 1 (and you can see all majors as well as minor rank 1 are quite close). I'll investigate more tomorrow.

**Comment by vituscze** [comment]
Fixed in 64dfd4c798baba898097420f0c6c5ff4294b25ed
